scripts/generate-docs-types-versions.js

```diff
@@ -0,0 +1,72 @@
+const fs = require("fs");
+const path = require("path");
+const { spawn } = require("child_process");
+const { promisify } = require("util");
+
+const mkdir = promisify(fs.mkdir);
+const cp = promisify(fs.copyFile);
+
+const work = (cwd, cmd, argv) =>
+  new Promise((resolve, reject) => {
+    const worker = spawn(cmd, argv, { cwd, shell: true });
+
+    worker.stdout.on("data", data => {
+      console.log(`stdout: ${data.toString()}`);
+    });
+
+    worker.stderr.on("data", data => {
+      console.log(`stderr: ${data.toString()}`);
+    });
+
+    worker.on("close", code => {
+      if (code) {
+        reject();
+      } else {
+        resolve();
+      }
+    });
+  });
+
+(async () => {
+  try {
+    await mkdir(path.resolve(__dirname, "tmp"));
+  } catch (error) {
+    console.log("failed creating directory tmp");
+    console.log(error);
+  }
+  const latest = "7.5.5";
+  const typesVersions = ["7.0.0", "7.2.0", "7.4.0", latest];
```

Comment:
We shouldn't hard-code this, otherwise we need to change it every time we release a new version.
I would prefer to always generate the docs in the `docs/types.md` file, using Babel's `master` branch. Maybe we coulld add a cli parameter to this script to use a local copy of Babel:
```
node scripts/generate-docs-types-versions.js --path ../babel/packages/babel-types
```